service/src/main/java/org/eclipse/keti/acs/policy/evaluation/cache/AbstractPolicyEvaluationCache.java

```diff
@@ -281,6 +279,7 @@ private void logSetEntityTimestampsDebugMessage(final String timestamp, final St
     @Override
     public void resetForPolicySet(final String zoneId, final String policySetId) {
         resetForEntity(zoneId, policySetId, EntityType.POLICY_SET, AbstractPolicyEvaluationCache::policySetKey);
+        resetForEntity(zoneId, StringUtils.EMPTY, EntityType.POLICY_SET, AbstractPolicyEvaluationCache::policySetKey);
```

Comment:
Instead of using an empty string key, its more explicit to define a constanct `POLICY_SET_ANY_KEY` with a value of `keti-any-policy-set-key`.

This constant can also be used by PolicyEvaluationServiceImpl to create PolicyEvaluationRequestCacheKey,  when the policySetEvaluationOrder is not specified.